### ISSUE TITLE
Webpack error "Can't find stylesheet to import." on _frontend.scss

### ISSUE THREAD

**Issue by Ruud-Zuiderlicht** [concrete5/bedrock#198]
I followed the instructions in https://documentation.concretecms.org/developers/appendix/concrete-cms-bedrock-foundation-concrete-and-concrete-themes to setup bedrock for a new theme. However, when I try to compile the scss-template  I get an import error from the bedrock file `@concretecms/bedrock/assets/bedrock/scss/_frontend.scss`:
```bash
@import '@fortawesome/fontawesome-free/scss/variables';
       ^
      Can't find stylesheet to import.
  ╷
5 │ @import '@fortawesome/fontawesome-free/scss/variables';
  │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

What I don't understand is that [line #5](https://github.com/concrete5/bedrock/blob/fc06fc11c58e073664c51d5ff30eb61aaa8c19e8/assets/bedrock/scss/_frontend.scss#L5) `@import '@fortawesome/fontawesome-free/scss/variables'`; is not understood (the actual file is called `_variables.scss`). But at [line #8](https://github.com/concrete5/bedrock/blob/fc06fc11c58e073664c51d5ff30eb61aaa8c19e8/assets/bedrock/scss/_frontend.scss#L8) `@import 'mixins'`; works perfectly fine (target file is called `_mixins.scss`). The only difference I see is that the former is a file from another module.

**On Windows or Ubuntu 20 these are the steps to reproduce**
- install concrete 9.0.2 (through composer)
- run `npm install`
- run `npm i laravel-mix` & `npm i @concretecms/bedrock`
- add `@import "~concretecms-bedrock/assets/bedrock/scss/frontend"; ` to the top of `resources/css/main.scss`
- as per the instructions include this in webpack.mix.js when you get the not found error (I did):
  ```
  mix.webpackConfig({
    module: {
        rules: [
            {
                resolve: {
                    alias: {
                        "concretecms-bedrock": "@concretecms/bedrock"
                    }
                }
            }
        ],
    },
  })
  ```
- run `npm run watch`

**This is the stack trace that Webpack outputs**
```
node_modules/@concretecms/bedrock/assets/bedrock/scss/_frontend.scss 5:9  @import
  stdin 20:9                                                                root stylesheet
      in /mnt/e/node_modules/@concretecms/bedrock/assets/bedrock/scss/_frontend.scss (line 5, column 9)
    at /mnt/e/node_modules/webpack/lib/NormalModule.js:313:20
    at /mnt/e/node_modules/loader-runner/lib/LoaderRunner.js:367:11
    at /mnt/e/node_modules/loader-runner/lib/LoaderRunner.js:233:18
    at context.callback (/mnt/e/node_modules/loader-runner/lib/LoaderRunner.js:111:13)
    at /mnt/e/node_modules/sass-loader/dist/index.js:89:7
    at Function.call$2 (/mnt/e/node_modules/sass/sass.dart.js:99012:16)
    at render_closure1.call$2 (/mnt/e/node_modules/sass/sass.dart.js:84527:12)
    at _RootZone.runBinary$3$3 (/mnt/e/node_modules/sass/sass.dart.js:29558:18)
    at _FutureListener.handleError$1 (/mnt/e/node_modules/sass/sass.dart.js:28080:21)
    at _Future__propagateToListeners_handleError.call$0 (/mnt/e/node_modules/sass/sass.dart.js:28387:49)
    at Object._Future__propagateToListeners (/mnt/e/node_modules/sass/sass.dart.js:3901:77)
    at _Future._completeError$2 (/mnt/e/node_modules/sass/sass.dart.js:28233:9)
    at _AsyncAwaitCompleter.completeError$2 (/mnt/e/node_modules/sass/sass.dart.js:27881:12)
    at Object._asyncRethrow (/mnt/e/node_modules/sass/sass.dart.js:3704:17)
    at /mnt/e/node_modules/sass/sass.dart.js:19778:20
    at _wrapJsFunctionForAsync_closure.$protected (/mnt/e/node_modules/sass/sass.dart.js:3729:15)
    at _wrapJsFunctionForAsync_closure.call$2 (/mnt/e/node_modules/sass/sass.dart.js:27900:12)
    at _awaitOnObject_closure0.call$2 (/mnt/e/node_modules/sass/sass.dart.js:27894:25)
    at _RootZone.runBinary$3$3 (/mnt/e/node_modules/sass/sass.dart.js:29558:18)
    at _FutureListener.handleError$1 (/mnt/e/node_modules/sass/sass.dart.js:28080:21)
    at _Future__propagateToListeners_handleError.call$0 (/mnt/e/node_modules/sass/sass.dart.js:28387:49)
    at Object._Future__propagateToListeners (/mnt/e/node_modules/sass/sass.dart.js:3901:77)
    at _Future._completeError$2 (/mnt/e/node_modules/sass/sass.dart.js:28233:9)
    at _AsyncAwaitCompleter.completeError$2 (/mnt/e/node_modules/sass/sass.dart.js:27881:12)
    at Object._asyncRethrow (/mnt/e/node_modules/sass/sass.dart.js:3704:17)
    at /mnt/e/node_modules/sass/sass.dart.js:15279:20
    at _wrapJsFunctionForAsync_closure.$protected (/mnt/e/node_modules/sass/sass.dart.js:3729:15)
    at _wrapJsFunctionForAsync_closure.call$2 (/mnt/e/node_modules/sass/sass.dart.js:27900:12)
    at _awaitOnObject_closure0.call$2 (/mnt/e/node_modules/sass/sass.dart.js:27894:25)
    at _RootZone.runBinary$3$3 (/mnt/e/node_modules/sass/sass.dart.js:29558:18)
    at _FutureListener.handleError$1 (/mnt/e/node_modules/sass/sass.dart.js:28080:21)
    at _Future__propagateToListeners_handleError.call$0 (/mnt/e/node_modules/sass/sass.dart.js:28387:49)
    at Object._Future__propagateToListeners (/mnt/e/node_modules/sass/sass.dart.js:3901:77)
    at _Future._completeError$2 (/mnt/e/node_modules/sass/sass.dart.js:28233:9)
    at _AsyncAwaitCompleter.completeError$2 (/mnt/e/node_modules/sass/sass.dart.js:27881:12)
    at Object._asyncRethrow (/mnt/e/node_modules/sass/sass.dart.js:3704:17)
 @ ./public/application/themes/ztheme/scss/framework.scss
```

**Comment by Ruud-Zuiderlicht** [comment]
A possible fix would be to change beforementioned [line #5](https://github.com/concrete5/bedrock/blob/fc06fc11c58e073664c51d5ff30eb61aaa8c19e8/assets/bedrock/scss/_frontend.scss#L5) to either on of these:
1. `@import '~@fortawesome/fontawesome-free/scss/variables';`
2. `@import '@fortawesome/fontawesome-free/scss/_variables.scss';`

**Comment by aembler** [comment]
I would highly recommend checking the new docs – I think those others are probably from before Bedrock was restructured a bit prior to 1.0 release. Please check this here instead https://documentation.concretecms.org/developers/pages-themes/designing-for-concrete5/building-theme-concrete-bedrock/introduction-bedrock

**Comment by Ruud-Zuiderlicht** [comment]
I think the old documentation is still ok. I followed your instructions on the new docs and got to a working situation. When comparing the two setups and found that relevant dependencies where not matching. So somehow the existing setup interfered and I ended up with outdated dependencies (even though they where first installed using the docs now). Below are the two modules that where outdated with the desired (correct) versions at the moment:

```json
"devDependencies": {
    "resolve-url-loader": "^5.0.0",
    "sass-loader": "^12.1.0",
}
```
